test/rules/cluster_capacity_cpu.yaml

```diff
@@ -161,5 +161,36 @@ tests:
         exp_samples:
           - labels: 'cluster:capacity_cpu_cores:sum{label_beta_kubernetes_io_instance_type="e2-standard-4",label_node_role_kubernetes_io="master",label_kubernetes_io_arch="amd64",label_node_openshift_io_os_id="rhcos"}'
             value: 5
+          - labels: 'cluster:capacity_cpu_cores:sum{label_beta_kubernetes_io_instance_type="e2-standard-4",label_kubernetes_io_arch="amd64",label_node_openshift_io_os_id="rhcos"}'
+            value: 4
+          - labels: 'cluster:capacity_cpu_cores:sum{label_beta_kubernetes_io_instance_type="e2-standard-4",label_node_role_kubernetes_io="infra",label_kubernetes_io_arch="amd64",label_node_openshift_io_os_id="rhcos"}'
+            value: 4
+  # node with both custom role ("ai-worker" in the test) is counted as worker.
```

Comment:
`node with both custom role` did you mean `node with a custom role other than infra or master`?

Reply:
exactly, let me rephrase it in a better way.